Headers/Public/VLCRendererDiscoverer.h

```diff
@@ -72,6 +72,11 @@
  */
 @property (nonatomic, readonly, copy) NSString * _Nonnull name;
 
+/**
+ * Renderers of the discoverer
+ */
+@property (nonatomic, readonly, copy) NSArray<VLCRendererItem *> * _Nonnull renderers;
```

Comment:
actually you can just add NS_ASSUME_NONNULL_BEGIN to the header and delete the nonnull everywhere and that will help with readability :) 
https://developer.apple.com/swift/blog/?id=25

Reply:
If I add `NS_ASSUME_NONNULL_BEGIN` wouldn't that mean that there will be multiple blocks since there are some element that are nullable such as the delegate?
I personally prefer stating explicitly that an element is `Nonnull` or `Nullable`.

Reply:
there won't be mutiple blocks it just assumes everything is nonnull unless stated differently